config/holes.toml

```diff
@@ -3034,3 +3034,51 @@ synopsis = 'Print 𝑒 to the first 1,000 decimal places.'
 preamble = '''
 <p>Print 𝑒 (Euler’s number) to the first 1,000 decimal places.
 '''
+['Rijndael S-boxes']
+ category = 'Number theory'
```

Comment:
That's not one of the existing categories.

Reply:
Changed to mathematics.